src/jit/lower.h

```diff
@@ -55,27 +55,63 @@ class Lowering : public Phase
             LowerNode(newNode);
         }
     }
+    void LowerRange(GenTree* firstNode, GenTree* lastNode)
+    {
+        for (GenTree* node = firstNode; node != lastNode->gtNext; node = node->gtNext)
+        {
```

Comment:
The Range constructor that takes first/last was private. I have now made it public and changed this to use it.

---
In reply to: [132026307](https://github.com/dotnet/coreclr/pull/13198#discussion_r132026307) [](ancestors = 132026307)